crates/rspack_plugin_javascript/src/dependency/commonjs/common_js_export_require_dependency.rs

```diff
@@ -132,7 +132,7 @@ impl DependencyTemplate for CommonJsExportRequireDependency {
             0
           )
         ),
-        None => "/* unused reexport */ ".to_string(),
```

Comment:
Can you add some `TODO:` comments, so that we could know this is just a workaround, we should align with webapck after the upstream dependency is finished.

Reply:
Comments had been added on lines 119 to 121.